test/integration/targets/ec2_transit_gateway/aliases

```diff
@@ -1,3 +1,4 @@
 cloud/aws
 shippable/aws/group2
 unstable  # sometimes tries to delete while in state pending
+ec2_transit_gateway_info
```

Comment:
Interesting, I didn't knew this can be done. I guess I have to look into that :)

Reply:
I've the same situation for other modules, that would improve global test performance somewhat.